pkgs/tools/networking/curl/default.nix

```diff
@@ -84,6 +87,8 @@ stdenv.mkDerivation rec {
     optional gsaslSupport gsasl ++
     optional gssSupport libkrb5 ++
     optional http2Support nghttp2 ++
+    optional http3Support nghttp3 ++
+    optional http3Support ngtcp2 ++
```

Comment:
This should be combined into a single line to make clear that the feature requires both packages.

```
    optionals http3Support [ nghttp3 ngtcp2 ] ++
```